<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameclass</class>
 <widget class="QMainWindow" name="gameclass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>LeaveGame</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif CJK JP</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>User:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif CJK JP</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif CJK JP</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Seconds left:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelQuestion">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif CJK JP</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Q:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelQuestionNumber">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif CJK JP</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif CJK JP</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Your choice:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelOptionPicked">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="RankingText">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>221</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="RankingLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>-10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ranking</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif CJK JP</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Points:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_points">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif CJK JP</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
